### ISSUE TITLE
Importer can't open database on Godot 3.2.2 (stable)

### ISSUE THREAD

**Issue by DarkPro1337** [thejustinwalsh/castledb-godot#4]
Importer can't open database and sends this error lines. Used **CastleDB 1.5** and **Godot 3.2.2.stable.official**.
This happens after plugin try to open the database after enabling it in **Project Settings**.

```
 res://addons/thejustinwalsh.castledb/castledb_utils.gd:43 - Invalid call. Nonexistent function 'back' in base 'PoolStringArray'.
 res://addons/thejustinwalsh.castledb/castledb_importer.gd:77 - Invalid operands 'String' and 'Nil' in operator '+'.
 core/os/file_access.cpp:667 - Condition "!f" is true. Continuing.
 Cannot open file 'res://.import/base.cdb-9ffa2a679c38d0a92db721a5abeeede5.gd'.
```

**Comment by thejustinwalsh** [comment]
Does it work with other databases? Can you send over your database, you are using?
There may have also been some updates to syntax and it is causing errors now. Need more details.

**Comment by DarkPro1337** [comment]
Here you go, I used this DB for my project. Now I use JSON parser of Godot, so don't hurry!
[base.zip](https://github.com/thejustinwalsh/castledb-godot/files/5126176/base.zip)

**Comment by thejustinwalsh** [comment]
Fixed in 4618f4a. Your CDB file contains a field `func` that is a reserved keyword in Godot however, if you change the name of the column `func` then you should be able to use the addon with that DB.
